### DISCUSSION TITLE
Is fusion planned for OpenStack?

### DISCUSSION THREAD

**Discussion by XaverStiensmeier** [nextflow-io/nextflow#3622] [General]
Hello,
we are currently considering moving away from our current file system and as we already use nextflow via **slurm** in a multi-cloud environment (single cloud working, multi-cloud in current development), using ignite (which is no longer developed) and now recently fusion is in discussion. As we are primarily working with OpenStack knowing whether OpenStack fusion support is planned would be of great interest (especially for planning) and of course any additional information on the timeline if it is planned.

Best regards,
Xaver Stiensmeier

**Comment by pditommaso** [comment]
Which nextflow executor you are planning to use in OpenStack? always ignite?

**Reply by XaverStiensmeier** [reply]
Thank you @pditommaso we are using slurm (in development: in a multi-cloud environment)

**Reply by pditommaso** [reply]
Support for Slurm has been merged [recently](https://github.com/nextflow-io/nextflow/pull/3516). I guess, you will need a S3-like object storage such as Minio for the storage layer

**Reply by XaverStiensmeier** [reply]
Interesting. Thank you. I am a bit irritated as the documentation states that fusion is only available for AWS https://www.nextflow.io/docs/latest/fusion.html
But that are great news. We will definitely take a closer look now.

Ah, it was very recent. Thank you very much.

**Reply by pditommaso** [reply]
The support Slurm and LSF has been added recently. The docs need to be updated
